drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -1143,9 +1190,12 @@ function DragAndDropBlock(runtime, element, configuration) {
     var updateDOM = function() {
         var new_vdom = render(state);
         var patches = virtualDom.diff(__vdom, new_vdom);
+        destroySlider();
```

Comment:
bxslider also offers a `reloadSlider` method -- would this work as a replacement for running destroy and then initialize again?

Reply:
Actually I used it deliberately, because in the next line I need root state when there is no bx slider wrappers on it(to compare it with old root version using virtualdom). Otherwise there are errors when comparing them.